Makefile

```diff
@@ -151,8 +155,24 @@ dbuild: crioimage
 integration: crioimage
 	$(CONTAINER_RUNTIME) run -e STORAGE_OPTIONS="--storage-driver=vfs" -e TEST_USERNS -e TESTFLAGS -e TRAVIS -t --privileged --rm -v ${CURDIR}:/go/src/${PROJECT} ${CRIO_IMAGE} make localintegration
 
-testunit:
-	$(GO) test -tags "$(BUILDTAGS) containers_image_ostree_stub" -cover $(PACKAGES)
+build-bin:
```

Comment:
should the rule be `${BUILD_BIN_PATH}/ginkgo` so we don't build it if it already exists?

Reply:
I was not sure how we want to handle this since the go compiler already supports some incremental build feature. So If I update some vendored deps the target would work out of the box. But if you prefer the usual Makefile style I can change it...

Reply:
`build-bin` makes me imply we're building the project's binary. Would you rename the target to `build-ginko` to avoid potential confusion?

Reply:
Alright, I will rename it.

Reply:
hm.. I still think we should avoid the extra build if not required

Reply:
Okay, then I will change it to `${BUILD_BIN_PATH}/ginkgo`